### PR TITLE
Prop to force the bash shell (disabled by default)

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/6029

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -49,6 +50,8 @@ Configuration DB format
 * ``DiscoverDcType {dns,ldapuri}``: ``dns`` query SRV records in AD DNS to find
   the DC name; ``ldapuri`` retrieve the DC name from the ``LdapURI`` prop value
 
+* ``ShellOverrideStatus`` (default ``disabled``) force the shell of all users to ``/bin/bash``.
```
Comment (DavidePrincipi) [inline README.rst]:
```suggestion
* ``ShellOverrideStatus`` (default ``disabled``) If ``enabled``, the value of the passwd user's shell field 
   is set to ``/bin/bash`` and is no more read from the accounts provider.
```

File: README.rst [hunk 2]
```diff
@@ -49,6 +50,8 @@ Configuration DB format
...
+  This property must be ``enabled`` on each server to force the shell access.
```
Comment (DavidePrincipi) [inline README.rst]:
```suggestion
```

**Comment by DavidePrincipi** [comment]
Why is it still a draft? ;)

**Comment by stephdl** [comment]
done @DavidePrincipi

File: createlinks [hunk 3]
```diff
@@ -49,6 +49,8 @@ event_templates('nethserver-sssd-save', qw(
     /etc/samba/smb.conf
     /etc/openldap/ldap.conf
     /etc/backup-config.d/nethserver-sssd.include
+    /etc/pam.d/cockpit
+    /etc/ssh/sshd_config
```
Comment (DavidePrincipi) [inline createlinks]: Do not expand templates of dependent RPMs!

Let's to it the other way round: individual packages, when installed, will bind the `nethserver-sssd-save` event to expand _their own_ template and restart _their own_ service.

File: createlinks [hunk 4]
```diff
@@ -58,6 +60,7 @@ event_actions('nethserver-sssd-save', qw(
 ));
 event_services('nethserver-sssd-save', qw(
     sssd restart
+    sshd restart
```
Comment (DavidePrincipi) [inline createlinks]: ...same as above.

File: createlinks (hunk 3 above)
Comment (stephdl) [reply createlinks]: ok, I am not sure to understand clearly, but let me advocate with another mind
What about if a sysadmin want to change the props `ShellOverrideStatus` to `enabled` manually, with only one evnt he can propagate his changes to any bound services ?

Comment (DavidePrincipi) [reply createlinks]: Yes, the procedure would be

    config setprop sssd ShellOverrideStatus enabled
    signal-event nethserver-sssd-save

The point here is: the responsibility of expanding templates and restarting services is up to nethserver-cockpit and nethserver-openssh, so they have to plug in their templates and service restart into `nethserver-sssd-save` event from their own `createlinks` script.

Comment (stephdl) [reply createlinks]: ok  it is clear, thank

Comment (stephdl) [reply createlinks]: done

File: createlinks (hunk 4 above)
Comment (stephdl) [reply createlinks]: done

**Pull request merged**